testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -717,6 +717,37 @@ The exact steps to configure Docker depend on the operating system.
 By default, the test will run against Undertow based embedded Keycloak Server, thus no distribution build is required beforehand.
 The exact command line arguments depend on the operating system.
 
+
+## Using DB Allocator Service
+
+The testsuite can use the DB Allocator Service to allocate and release desired database automatically.
+Since some of the database properties (such as JDBC URL, Username or Password) need to be used when building the Auth Server,
+the allocation and deallocation need to happen when building the `integration-arquillian` project (instead of `tests/base` as
+it happens in other cases).
+
+In order to use the DB Allocator Service, you must use the `jpa` profile with one of the `db-allocator-*`. Here's a full example to
+run JPA with Auth Server Wildfly and MSSQL 2016:
+
+```
+mvn -f testsuite/integration-arquillian/pom.xml \
+    -Pjpa,auth-server-wildfly,db-allocator-db-mssql2016 \
+    -Ddballocator.uri=<<db-allocator-servlet-url>> \
+    -Ddballocator.user=<<db-allocator-user>> \
+    -Dmaven.test.failure.ignore=true
+```
+
+Using `-Dmaven.test.failure.ignore=true` is not strictly required but highly recommended. After running the tests,
+the DB Allocator Plugin should release the allocated database.
+
+Other implemented profiles:
```

Comment:
Since this will be changing over time, I suggest to put one or two profiles here and refer user to the output of the following command to get the list of available db profiles:

```bash
mvn help:all-profiles -pl testsuite/integration-arquillian | grep -- db-
```

Reply:
hmmmm it would look inconsistent with the rest of the writing in `test-db.md`. Let me leave it like it is now.

Reply:
Fixed, thanks!